cou/cli.py

```diff
@@ -137,8 +138,18 @@ async def get_upgrade_plan(args: CLIargs) -> None:
     :param args: CLI arguments
     :type args: CLIargs
     """
-    upgrade_plan = await analyze_and_plan(args)
+    upgrade_plan, warning_messages = await analyze_and_plan(args)
     print_and_debug(upgrade_plan)
+
+    if warning_messages:
+        for warning in warning_messages:
+            logger.warning(warning)
+        print(
+            "Running upgrades will not be possible until problems indicated in the warnings "
+            "are resolved."
+        )
```

Comment:
Should this be an error log message instead? Or maybe even better would be raising an exception here?

Reply:
I don't think we should raise an exception here, but I agree with the one in `run_upgrade` bellow